### ISSUE TITLE
Your GPDPR Quote is incomplete / wrongly applied.

### ISSUE THREAD

**Issue by HealsCodes** [HorizonFFXI/hxiloader#1]
Yes, a MAC address on it's own is only a machine identifier and not personal data.

_However_ you are linking that MAC address to an account in order to achieve the goal of identifying cheaters - see where this goes? This creates a link between a person and a device... which is no longer just a machine identifier.

There's no means in the GPDR to identify an account based off hardware data that is not to some extend reaching into the "personally identifiable" area. 

The safest thing you could do is disconnect an active client from the server if another client - regardless of the account data - attempts to connect with the same MAC address. In that case you would have no idea who's account it was and would be able to state so in full compliance with GDPR.

**Comment by 0phie** [comment]
Hi there! 

Sorry, we don't check issues on this repo all that often. We have already consulted with a privacy lawyer regarding this exact scenario and were informed that we are still in full compliance with GDPR under the current verbiage and usage of the bootloader's gathering of MAC addresses. By release we will release our full privacy policy with an exact explanation of the data we collect, how to contact us, and the process for deleting that data.

I'll close this for now if you have any other questions, however as stated we have looked into this issue and did get confirmation from someone who is better versed in law than a bunch of engineers. :)
